src/UXClient/Components/LineChart/LineChart.ts

```diff
@@ -1608,6 +1621,9 @@ class LineChart extends TemporalXAxisComponent {
                                 yExtent = self.swimlaneYExtents[agg.swimLane];
                             }
 
+                            // Update yExtent in AggExpOpts to reflect current yExtent
+                            if(self.aggregateExpressionOptions[i].yExtent === null) self.aggregateExpressionOptions[i].yExtent = yExtent;
```

Comment:
@niusoff this is what Matt wanted me to sync with you about.  How does this look.  

The idea is to show the d3 calculated yExtent in aggregateExpression options (even when no custom yExtent is given).  This allows the T7 settings modal to display the auto-generated extents.